LocalDynamoDb/LocalDynamo.cs

```diff
@@ -70,7 +71,7 @@ public void Stop()
             {
                 Dynamo.Kill();
             }
-            catch (Win32Exception)
+            catch (Exception)
```

Comment:
There's probably a better exception type to catch that's xplat, though it wouldn't surprise me if maybe `Win32Exception` can still happen even on not-Windows.

Reply:
Yup! Looks like it does now I've had a chance to try it on my Macbook, will put it back.